datastore/integration_test.go

```diff
@@ -739,20 +742,75 @@ func TestIntegration_AggregationQueries(t *testing.T) {
 	}{
 		{
 			desc:          "Count Failure - Missing index",
-			aggQuery:      baseQuery.Filter("T>=", now).NewAggregationQuery().WithCount("count"),
+			aggQuery:      NewQuery("SQChild").Ancestor(parent).Filter("T>=", now).NewAggregationQuery().WithCount("count"),
 			wantFailure:   true,
 			wantErrMsg:    "no matching index found",
 			wantAggResult: nil,
 		},
 		{
 			desc:        "Count Success",
-			aggQuery:    baseQuery.Filter("T=", now).Filter("I>=", 3).NewAggregationQuery().WithCount("count"),
+			aggQuery:    NewQuery("SQChild").Ancestor(parent).Filter("T=", now).Filter("I>=", 3).NewAggregationQuery().WithCount("count"),
 			wantFailure: false,
 			wantErrMsg:  "",
 			wantAggResult: map[string]interface{}{
 				"count": &pb.Value{ValueType: &pb.Value_IntegerValue{IntegerValue: 5}},
 			},
 		},
+		{
+			desc:        "Multiple aggregations",
+			aggQuery:    NewQuery("SQChild").Ancestor(parent).Filter("T=", now).NewAggregationQuery().WithSum("I", "i_sum").WithAvg("I", "avg").WithSum("V", "v_sum"),
+			wantFailure: false,
+			wantErrMsg:  "",
```

Comment:
style nit: would it be better to omit this and rely on the zero value?

Reply:
Done